stor/tests/test_dx.py

```diff
@@ -1197,7 +1231,7 @@ def test_dx_to_dx_within_project_pass(self):
                           '/another_folder/random_file.txt'])
         dx_p = DXPath('dx://' + self.project + ':/temp_folder/file.txt')
         new_dx_p = DXPath('dx://' + self.project + ':/another_folder/fold_file.txt')
-        dx_p.copy(new_dx_p, move_within_project=True)
+        stor.copy(dx_p, new_dx_p, move_within_project=True)
```

Comment:
can you remove the `move_within_project=True` here just to confirm that's the default?

Reply:
Yep, I changed the default and redid the tests.